examples/testplans/release/release.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "Release Plan",
+    "description": "Release Plan to be performed during the release itself",
+
+    "tests": [
+         {
+	 "name": "Version Bump: Version number",
+         "description": "For the Avocado versioning, two files need to receive a manual version update: `VERSION` and `python-avocado.spec`."
+	 },
+         {
+	 "name": "Version Bump: Propagate to plugins",
+         "description": "Run the command: `make propagate-version` to propagate this change to all optional and “linkabe” plugins sharing the parent dir (eg. avocado-vt)."
+	 },
+         {
+	 "name": "Version Bump: Propagate to plugins",
+         "description": "Run the command: `make propagate-version` to propagate this change to all optional and “linkabe” plugins sharing the parent dir (eg. avocado-vt)."
+	 },
+         {
+	 "name": "Version Bump: Commit the bump",
```

Comment:
The commit should also include a new release notes document, so this is probably a new step.

Reply:
Sorry, @clebergnu I didn't understand you. Can you explain what you mean, please?

Reply:
What I mean can be seen by `git show --stat 73.0`:

```
...
 VERSION                                        |  2 +-
 docs/source/releases/73_0.rst                  | 97 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 docs/source/releases/index.rst                 |  1 +
...
```

That is, a release notes document (for the new version) should be created/added.

Reply:
hum ok, understood now. I will add a new step.